packages/wonder-blocks-tokens/src/theme/primitive/font.ts

```diff
@@ -78,4 +78,8 @@ export const font = {
             xxlarge: sizing.size_400, // Title
         },
     },
+    textDecoration: {
```

Comment:
Nice! Thanks for sorting out the new tokens! 😄 

Do we want to include the term `underline` somewhere in the token name? Related to [Caitlyn's comment](https://khanacademy.slack.com/archives/C07CA2YR0BZ/p1755005715448969?thread_ts=1754955327.064349&cid=C07CA2YR0BZ)

Reply:
Yes, updated!